pipeline.py

```diff
@@ -59,30 +81,71 @@ class TextProcessingPipeline:
     Process articles from corpus manager
     """
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
+        self.text_to_process = ''
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles()
+        for file in articles.values():
+            self.text_to_process = file.get_raw_text()
+            morph_tokens = self._process()
+            processed_text = []
+            for token in morph_tokens:
+                processed_text.append(str(token))
```

Comment:
use list comprehension at least. can you think of a more beautiful **functional** solution? 

hint: remember map, filter, reduce functions?..